apps/cow-tools/src/app/milkman/components/TransactionProgressBar.tsx

```diff
@@ -0,0 +1,132 @@
+"use client";
+
+import cn from "clsx";
+import { PropsWithChildren } from "react";
+
+import { TransactionStatus } from "../utils/type";
+
+type Stage = {
+  name: TransactionStatus;
+  stepNumber: number;
+  defaultStepClass: string;
+  activeStepClass: string;
+  completedStepClass: string;
+  defaultLineColor: string;
+  activeLineColor: string;
+  completedLineColor: string;
+  nextStage?: TransactionStatus;
+  previousStage?: TransactionStatus;
+};
+
+export const stages: Stage[] = [
+  {
+    name: TransactionStatus.DRAFT_SELECT_TOKENS,
+    stepNumber: 1,
+    defaultStepClass: "border-slate10 text-slate12",
+    activeStepClass: "border-amber10 text-amber10",
+    completedStepClass: "border-green10 text-green10",
+    defaultLineColor: "bg-slate10",
+    activeLineColor: "bg-amber10",
+    completedLineColor: "bg-green10",
```

Comment:
these properties seem to be constants for all stages. If so, I would try to avoid code repetition